libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -22,6 +22,7 @@ import { WizardProgressBarDirective } from './wizard-progress-bar/wizard-progres
 import { scrollTop } from '@fundamental-ngx/core/utils';
 import { ACTIVE_STEP_STATUS, CURRENT_STEP_STATUS, UPCOMING_STEP_STATUS, COMPLETED_STEP_STATUS } from './constants';
 import { DialogBodyComponent } from '@fundamental-ngx/core/dialog';
```

Comment:
Could you please move this import up, so you import from other libraries before you import from local library